app/react/Markdown/utils.js

```diff
@@ -13,3 +13,121 @@ export const logError = (err, propValueOf, propLabelOf) => {
   console.error(err);
   /* eslint-enable no-console */
 };
+
+export const validHtmlTags = new Set([
```

Comment:
As you mentioned, iframes will not work since they are not considered valid. I don't have an opinion on whether we should include them or not. I do think a user might want to embed a video from another platform in a custom page, and would not be able to without iframes.

Reply:
We have decided to allow svg and iframe.